spec/unit/actions/synchronize_etls_in_segment_spec.rb

```diff
@@ -7,21 +7,52 @@
 require 'gooddata/lcm/lcm2'
 require 'gooddata/lcm/actions/synchronize_etls_in_segment'
 
+def process_mock(name, id)
+  mock = double(GoodData::Process)
+  allow(mock).to receive(:name) { name }
+  allow(mock).to receive(:process_id) { id }
+  mock
+end
+
+def schedule_mock(name, process_id)
```

Comment:
warning: Similar code found in synchronize_etls_in_segment_spec.rb:10, synchronize_etls_in_segment_spec.rb:17 (mass = 42) (flay)